tests/unit/test_conversion.py

```diff
@@ -0,0 +1,30 @@
+"""Test that configurations properly convert from undesired format to desired"""
+import glob
+import os
+
+import pytest
+from netutils.config.conversion import (
+    paloalto_panos_brace_to_set,
+)
+from netutils.config.compliance import conversion_map
+
+MOCK_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "mock", "config", "conversion")
+
+TXT_FILE = "_sent.txt"
+CONVERTED_FILE = "_converted.txt"
+
+conversion_files = []
+
+for network_os in list(conversion_map.keys()):
```

Comment:
Jeff pointed out that `config/conversion/` is defined both in the `MOCK_DIR` and in the `glob` line of code. Likely it should just be: 

```python
    for _file in glob.glob(f"{MOCK_DIR}/{network_os}/*{TXT_FILE}"):
        conversion_files.append([_file, network_os])
```


```
tests/unit/test_conversion.py::test_config_conversion[_file0] SKIPPED (got empty parameter set ['_file'], function test_config_conversion at /local/tests/unit/test_conversion.py:22) [ 34%]
```